products/batch_exports/backend/temporal/pipeline/consumer.py

```diff
@@ -41,85 +43,49 @@ def bytes_exported_counter(self) -> temporalio.common.MetricCounter:
         """Access the bytes exported metric counter."""
         return get_bytes_exported_metric()
 
+    def reset_tracking(self) -> None:
+        self.total_record_batches_count = 0
+        self.total_records_count = 0
+        self.total_record_batch_bytes_count = 0
+        self.total_file_bytes_count = 0
+
     async def start(
         self,
         queue: RecordBatchQueue,
         producer_task: asyncio.Task,
-        schema: pa.Schema,
-        transformer: TransformerProtocol,
-        max_file_size_bytes: int = 0,
+        transformer: ChunkTransformerProtocol,
         json_columns: collections.abc.Iterable[str] = ("properties", "person_properties", "set", "set_once"),
```

Comment:
The plan is to get rid of this in the very near future. In fact, I tried in this PR, but we rely on some conversions happening in `JsonScalar` that need to be properly accounted for, so I left it out to reduce the scope of this PR.